app/controllers/registrations_controller.rb

```diff
@@ -0,0 +1,12 @@
+class RegistrationsController < Devise::RegistrationsController
```

Comment:
There is no need to create a registration_controller.rb 
As in Device documentation add some code in existing ApplicationController (https://github.com/plataformatec/devise)

```
class ApplicationController < ActionController::Base
  before_action :configure_permitted_parameters, if: :devise_controller?

  protected

  def configure_permitted_parameters
    devise_parameter_sanitizer.permit(:sign_up, keys: [:name])  
    devise_parameter_sanitizer.permit(:account_update, keys: [:name])
  end
end
```
Only :name field must be required not ``` :email, :name, :password, :password_confirmatio ```